libs/api.lua

```diff
@@ -168,6 +209,9 @@ return function (db, prefix)
   end
 
   local routes = {
+    "^/metrics$", function (hash, path)
```

Comment:
no need for the `hash` and `path` arguments here, there are no pattern matches in this route.